client/containers/Pub/SpubHeader/SpubHeaderToolBar/spubHeaderToolbar.scss

```diff
@@ -0,0 +1,40 @@
+@import 'styles/variables.scss';
+
+.spubheader-toolbar {
+	display: flex;
+	justify-content: center;
+	align-items: baseline;
+	column-gap: 340px;
+	.tabs.spub-header-component .bp3-tab-list {
+		margin: 0px;
+	}
+	.spub-header-component .bp3-tab-list .bp3-tab.inactive {
+		opacity: 0.45;
+		color: var(--community-accent-dark);
+	}
+}
+
+.gridParent {
+	&.container {
```

Comment:
I believe we want to keep the `width: 96%` here to keep things in line with other grid elements. However, we do also want to add this:

```css
.col-12 {
    margin-top: 0;
    margin-bottom: 0;
}
```
to remove the excess vertical margin around one of the `GridWrapper` children. Apologies that this is so fussy — we will eventually migrate off of `GridWrapper`.